### PR TITLE
Code cleanup of Sitting module

### PR DESCRIPTION
**Pull request by MikeMatrix**
Code cleanup of the Sitting module.

### CODE REVIEW

File: addons/sitting/functions/fnc_getRandomAnimation.sqf [hunk 1]
```diff
@@ -46,4 +46,4 @@ _animations = [
 ];
 
 // Select random animation
-(_animations select (floor (random (count _animations))))
+(_animations select floor random count _animations)
```
Comment (Glowbal) [inline addons/sitting/functions/fnc_getRandomAnimation.sqf]: I am not certain if this is improving the readability - it doesn't matter for performance so I'd suggest just leaving those in.

Comment (MikeMatrix) [inline addons/sitting/functions/fnc_getRandomAnimation.sqf]: It doesn't do anything for performance. It's just a personal preference to keep the amounts of required parenthesis to a minimum whenever I can, considering they often lead to a confusing looking syntax.

**Comment by jonpas** [comment]
Looks good to me, I will be doing fixes for sitting (changing from PFH to CfgMoves config), so I'll see if there is any problem when I do that, good to merge from my side.

**Comment by MikeMatrix** [comment]
@bux578 you have no idea how close I was to making the commit message "Code Cleanup - The Return of the Parenthesis"... :shipit:

**Comment by bux578** [comment]
lol

**Comment by Glowbal** [comment]
Everything looks ok. My brain isn't functioning good enough at the moment to confirm the changes to `fn_hasChairMoved.sqf` are all correct though.

**Comment by jonpas** [comment]
Tested, works well, `FUNC(hasChairMoved)` update is good! :+1:

**Comment by Glowbal** [comment]
Thanks @jonpas

**Pull request merged**